File: Backend/TopSys.TopConWeb.Domain/Entities/LiberacaoAcesso/GrupoAcesso.cs
using System;
using System.Collections.Generic;

namespace TopSys.TopConWeb.Domain.Entities.LiberacaoAcesso
{
    public class GrupoAcesso
    {
        public int Codigo { get; set; }

        public int Usina { get; set; }

        public string Descricao { get; set; }

        public DateTime? CriadoEm { get; set; }

        public DateTime? AtualizadoEm { get; set; }

        public virtual ICollection<LiberacaoAcesso> LiberacoesAcessos { get; set; }
    }
}

File: Frontend/src/app/classes/acesso-aprovacoes/grupo-acesso.ts
import { CadastroDiverso } from "../cadastro-geral/cadastro-diverso";
import { Usina } from "../usina/usina";
import {LiberacaoAcesso } from "./liberacao-acesso";

export class GrupoAcesso {
    codigo: number = 0;
    usina: number = 0;
    descricao: string = '';
    criadoEm: Date = new Date();
    atualizadoEm: Date = new Date();
    liberacoesAcessos: LiberacaoAcesso[] = [];
    usuarios: CadastroDiverso[] = [];
    usinaInf: Usina;
}